qcodes/tests/dataset/test_dataset_loading.py

```diff
@@ -178,3 +184,16 @@ def test_get_data_by_id_order(dataset):
     data_dict = {el['name']: el['data'] for el in data[1]}
     assert data_dict['indep1'] == 1
     assert data_dict['indep2'] == 2
+
+
+@pytest.mark.usefixtures('experiment')
+def test_load_by_guid(some_paramspecs):
+        paramspecs = some_paramspecs[2]
+        ds = DataSet()
+        ds.add_parameter(paramspecs['ps1'])
+        ds.add_parameter(paramspecs['ps2'])
+        ds.add_result({'ps1': 1, 'ps2': 2})
+
+        loaded_ds = load_by_guid(ds.guid)
+
+        assert loaded_ds.the_same_dataset_as(ds)
```

Comment:
unintentional extra 4 spaces of indentation here?

Reply:
Yup.